zenguard_library.ipynb

```diff
@@ -0,0 +1,184 @@
+            "Requirement already satisfied: numpy>=1.22.4 in /usr/local/lib/python3.10/dist-packages (from pandas<3.0.0,>=2.2.1->zenguard) (1.25.2)\n",
+            "Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.10/dist-packages (from pandas<3.0.0,>=2.2.1->zenguard) (2.8.2)\n",
+            "Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.10/dist-packages (from pandas<3.0.0,>=2.2.1->zenguard) (2023.4)\n",
+            "Requirement already satisfied: tzdata>=2022.7 in /usr/local/lib/python3.10/dist-packages (from pandas<3.0.0,>=2.2.1->zenguard) (2024.1)\n",
+            "Requirement already satisfied: exceptiongroup in /usr/local/lib/python3.10/dist-packages (from anyio->httpx<1.0.0,>=0.24.0->zenguard) (1.2.1)\n",
+            "Requirement already satisfied: annotated-types>=0.4.0 in /usr/local/lib/python3.10/dist-packages (from pydantic<3,>=1.9.0->openai<2.0.0,>=1.14.2->zenguard) (0.7.0)\n",
+            "Requirement already satisfied: pydantic-core==2.18.2 in /usr/local/lib/python3.10/dist-packages (from pydantic<3,>=1.9.0->openai<2.0.0,>=1.14.2->zenguard) (2.18.2)\n",
+            "Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.10/dist-packages (from python-dateutil>=2.8.2->pandas<3.0.0,>=2.2.1->zenguard) (1.16.0)\n",
+            "Installing collected packages: rapidfuzz, h11, pandas, httpcore, httpx, openai, zenguard\n",
+            "  Attempting uninstall: pandas\n",
+            "    Found existing installation: pandas 2.0.3\n",
+            "    Uninstalling pandas-2.0.3:\n",
+            "      Successfully uninstalled pandas-2.0.3\n",
+            "\u001b[31mERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.\n",
+            "cudf-cu12 24.4.1 requires pandas<2.2.2dev0,>=2.0, but you have pandas 2.2.2 which is incompatible.\n",
+            "google-colab 1.0.0 requires pandas==2.0.3, but you have pandas 2.2.2 which is incompatible.\u001b[0m\u001b[31m\n",
+            "\u001b[0mSuccessfully installed h11-0.14.0 httpcore-1.0.5 httpx-0.27.0 openai-1.30.2 pandas-2.2.2 rapidfuzz-3.9.1 zenguard-0.1.13\n"
+          ]
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "### Configure API key:\n",
+        "1. Navigate to the [Settings](https://console.zenguard.ai/settings)\n",
+        "2. Click on the **+ Create new secret key**.\n",
+        "3. Name the key **Quickstart Key**.\n",
+        "4. Click on the **Add** button.\n",
+        "5. Copy the key value by pressing on the copy icon.\n"
+      ],
+      "metadata": {
+        "id": "WoJ6aXR46CCt"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "ZENGUARD_API_KEY=\"\""
```

Comment:
can we setup API key from OS env? instead of having it being displayed in the open as it is now?